src/passes/Print.cpp

```diff
@@ -454,7 +454,7 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
     if (curr->params.size() > 0) {
       for (size_t i = 0; i < curr->params.size(); i++) {
```

Comment:
The usual`for` idiom has array indexing within it. In this case there is none. So I could go either way. The toIndex() is needed somewhere because in 64-bit, sizeof(size_t) > sizeof(Index).